kerl

```diff
@@ -2216,6 +2249,9 @@ upgrade_kerl() {
     notice "kerl $version is now available at $current_kerl_path."
 }
 
+fix_otp_builds
+fix_otp_installations
+
```

Comment:
Putting this here _might be_ overkill, but I fear we might put it in e.g. `kerl delete ...` then introduce a new command and forget to copy it there, in the future.

On the other hand, this is a rare occurrence (that the files are out of sync), I imagine, so probably it'd be Ok to have it in specific cases alone.

... but it should be a simple/fast operation most of the time.

Reply:
I was contemplating adding [a custom command](https://asdf-vm.com/plugins/create.html#extension-commands-for-asdf-cli) to [`asdf-erlang`](https://github.com/asdf-vm/asdf-erlang) named something like `asdf erlang doctor` that users can run to manually cleanup the contents of `KERL_BASE_DIR` ([which `asdf-erlang` sets here](https://github.com/asdf-vm/asdf-erlang/blob/master/lib/utils.sh#L46-L54)).

I don't know if that makes more sense to live in `kerl` itself and have `asdf-erlang` simply wrap it, or if it makes sense to live fully in`asdf-erlang`.

Thoughts?

cc @Stratus3D 😄

Reply:
It's probably best this is discussed in a separate issue, but:

1. what would that command be for?
2. apart from this recently added "cleanup fix", I'm not sure `kerl` does any other `doctor` -related stuff (I can imagine a command like it existing, to report on "broken" stuff, but not sure if it's in `kerl`'s immediate plans - maybe the new issue can help focus on this?)
3. `kerl` already has command `cleanup` (not sure this is considered a `doctor` action, but you did mention "cleanup the contents of", so...)

Reply:
Ah, I didn't realize that `kerl` has `cleanup`, thanks! Though [apparently `asdf-erlang` already uses it but it might not be working currently](https://github.com/asdf-vm/asdf-erlang/issues/280), which would corroborate what I've noticed as well. I'll talk about that in the aforementioned issue.

RE putting `fix_otp_builds` and `fix_otp_installations` on every call to `kerl`: I don't think it's overkill. Your "forget to copy it there" fear is valid. 😄

Reply:
Yes, I think the proper solution for asdf-erlang is to fix that issue in asdf-erlang. For this PR do whatever is best for kerl, asdf-erlang will be updated as necessary! I'll investigate https://github.com/asdf-vm/asdf-erlang/issues/280 this week.

Reply:
Resolve'ing as preparing to merge.